mipidsi/examples/spi-st7789-rpi-zero-w/README.md

```diff
@@ -0,0 +1,30 @@
+# SPI ST7789 on a Raspberry Pi Zero W Example
+
+Tested with [Display HAT Mini by Pomoroni](https://shop.pimoroni.com/products/display-hat-mini?variant=39496084717651).
+
+## Build, strip, copy, and run
+
+### Mac OS
+
+Prerequisite: musl-cross (Homebrew users: `brew install filosottile/musl-cross/musl-cross`)
+
+```bash
+# build for rpi zero 2 w
```

Comment:
Good catch. The comment is wrong, I meant `rpi zero w` which is the one I'm testing on. Version 2 is ARMv8-A (64/32-bit), while version 1 is ARMv6Z (32-bit). Sadly I can't test version 2, but from what I've read [here](https://github.com/japaric/rust-cross) it may work but won't include optimizations, due to be using `arm` in the triple instead of a more specific one like `armv6`. I'm going to fix the comment